HW3/task6.py

```diff
@@ -0,0 +1,5 @@
+a = 2
+b = 5
+c = 6
+d = (str(a) + str(b) + str(c))
```

Comment:
nit: no need to use brackets here
```suggestion
d = str(a) + str(b) + str(c)
```